src/View/ThemeResourceLoader.php

```diff
@@ -174,6 +181,12 @@ public function getPath($identifier)
      */
     public function findTemplate($template, $themes = null)
```

Comment:
Can we add a unit test scenario where we call `findTemplate` twice in a row with the same arguments to make sure the method works correctly when it hits the cache?